zerver/openapi/zulip.yaml

```diff
@@ -2324,12 +2327,16 @@ paths:
                                     [
                                       {
                                         "type": "hotspot",
-                                        "name": "intro_streams",
-                                        "title": "Catch up on a stream",
-                                        "description": "Messages sent to a stream are seen by everyone subscribed to that stream. Try clicking on one of the stream links below.",
+                                        "name": "intro_compose",
+                                        "title": "Compose",
+                                        "description": "Click here to start a new conversation. Pick a topic (2-3 words is best), and give it a go!",
                                         "delay": 0.5,
                                         "has_trigger": false,
                                       },
+                                      {
+                                        "type": "one_time_notice",
+                                        "name": "visibility_policy_banner",
+                                      },
```

Comment:
Seemed useful to have an example of these one-time notices as well.

Reply:
Yeah, good idea, especially since we're deleting the hotspots soon.